fplus-http-server/src/router/allocator.rs

```diff
@@ -100,6 +100,13 @@ pub async fn create_from_json(
                 } else {
                     Some(model.application.tooling.join(", "))
                 };
+
+                let disable_ssa_bot = if model.application.disable_ssa_bot.is_none() {
+                    Some(false)
+                } else {
+                    model.application.disable_ssa_bot
+                };
```

Comment:
```suggestion
                let disable_ssa_bot = model.application.disable_ssa_bot.or(Some(false));
```
https://doc.rust-lang.org/std/option/enum.Option.html#method.or